packages/domain-picker/src/domain-picker/use-your-domain-item.tsx

```diff
@@ -22,12 +22,20 @@ const UseYourDomainItem: React.FunctionComponent< Props > = ( { onClick } ) => {
 				</span>
 				<div>
 					<span className="domain-picker__item-tip">
-						{ __( "You can use it as your site's address", __i18n_text_domain__ ) }
+						{ _x(
```

Comment:
Out of curiosity, why is the context needed when translating this piece of text?

Reply:
When `context` is present, it needs to be matching as well as the original string in order to re-use the [existing translation](https://translate.wordpress.com/projects/wpcom/-all-translations/234678/). Otherwise you just create a [new entry](https://translate.wordpress.com/projects/wpcom/-all-translations/631417/).
FG for more details:  https://fieldguide.automattic.com/i18n/developers/context-vs-comment/

Reply:
Thank you! I'm going to read the FG entry